pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -375,18 +375,19 @@ func (lb *ListenerBuilder) buildSidecarOutboundListeners(node *model.Proxy,
 					egressListener.IstioListener.Port.Number, node.ID)
 				continue
 			}
-			if conflictWithStaticListener(node, int(egressListener.IstioListener.Port.Number)) {
-				log.Warnf("buildSidecarOutboundListeners: skipping sidecar port %d for node %s as it conflicts with static listener",
-					egressListener.IstioListener.Port.Number, node.ID)
-				continue
-			}
 
 			listenPort := &model.Port{
 				Port:     int(egressListener.IstioListener.Port.Number),
 				Protocol: protocol.Parse(egressListener.IstioListener.Port.Protocol),
 				Name:     egressListener.IstioListener.Port.Name,
 			}
 
+			if listenPort.Protocol.IsHTTP() && conflictWithStaticListener(node, listenPort.Port) {
```

Comment:
Can you add comments on why only Http is considered?

Reply:
Should we move that logic to conflictWithStaticListener?

Reply:
>Can you add comments on why only Http is considered?

Because if this is TCP, we generate listeners with "vip_15090", it will not conflict with static listener

Reply:
>Should we move that logic to conflictWithStaticListener?

Because this function is also used by inbound listener building

Reply:
>Because if this is TCP, we generate listeners with "vip_15090", it will not conflict with static listener

Please add it in the comments